src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrderStaticGroupSeperated.java

```diff
@@ -0,0 +1,13 @@
+package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
+
+import static java.lang.Math.abs;
+
+import static java.lang.Math.cos;;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
@djydewang Did you see my top level comment in the previous review? https://github.com/checkstyle/checkstyle/pull/5091#pullrequestreview-62893844
> In InputImportOrderStaticGroupSeperated please add another group of imports with a comment between 2 imports with no blank lines between them.

Reply:
yean,I have fixed it before.Maybe I changed it for test later and forget to fix it.
I will fix it.

Reply:
@rnveach  Did you see my last comment?

Reply:
I see it now. Thanks.